### DISCUSSION TITLE
I don't see any comment over the generated MessageBundleLocalizedStrings

### DISCUSSION THREAD

**Discussion by Curwe** [comahe-de/i18n4k#61] [General]
Android kotlin project.
v0.7.1 core-jvm

Hi.
I've used this library back in the day to generate strings and back then we get a comment above each generated val to give a hint on what string it return. But I've added this lib to a new project and all my generated files are missing the comment. It make sit hard to know what I'm looking at. I might have missed something during setup. 

I tried going back to v0.5.0 that I used before but I still didn't see any comments. 
I did see that you have comments in the README also, would be nice if you could help me here :) 

This is what it looks like for me when I generate my files.

```
public object PhraseStrings : MessageBundle("PhraseStrings", "") {

  @JvmStatic
  public val account_app_settings_menu_notifications_status_off: MessageBundleLocalizedString =
      getLocalizedString0("account_app_settings_menu_notifications_status_off", 0)

  @JvmStatic
  public val account_app_settings_menu_notifications_status_on: MessageBundleLocalizedString =
      getLocalizedString0("account_app_settings_menu_notifications_status_on", 1)
      
      ....
}
```

Gradle 

```
i18n4k {
    inputDirectory = translation.downloadDir // path to *.properties files download in a different script.
    sourceCodeLocales = listOf("en_GB", "sv_SE")
}

android {
.... 
}
```

Thanks

**Comment by comahe-de** [comment]
There is a variable named `commentLocale` in the Gradle configuration object `i18n4k`. 

It has the default value "en". Make sure that you have the locale "en" or the configured one. 

"en_GB" will not default to "en"!

**Reply by Curwe** [reply]
Was just about to comments that I found exactly that. Would it be possible in for a future release to have it mutable so we can set commentLocale ourself or Maybe have it take the first value in `sourceCodeLocales`? 

Thanks tho!

**Reply by comahe-de** [reply]
As I said, you can configure it in the Gradle configuration object `i18n4k`.

```
i18n4k {
    commentLocale  = "en_GB"
}
```

**Comment by Curwe** [comment]
I con't seem to get it to work tho. 
I'm using gradle.kts if that would make a difference. 

![Screenshot 2024-03-15 at 10 14 57](https://github.com/comahe-de/i18n4k/assets/135611838/1e879853-e1b0-4ebc-b020-d2f49157279c)

**Reply by comahe-de** [reply]
Oh. That's a bug. I used "val" instead of "var" in den code.

It will be changed in the next release!

**Reply by Curwe** [reply]
Perfect thanks a lot then! I'm doing a work around for now with both a en and a en_GB then :)

**Reply by comahe-de** [reply]
OK.

Issue created #62
